src/test/scala/org/ergoplatform/mining/ErgoMinerSpec.scala

```diff
@@ -221,15 +167,17 @@ class ErgoMinerSpec extends FlatSpec with ErgoTestHelpers with ValidBlocksGenera
     val prop1: DLogProtocol.ProveDlog = DLogProverInput(BigIntegers.fromUnsignedByteArray("test1".getBytes())).publicImage
     val prop2: DLogProtocol.ProveDlog = DLogProverInput(BigIntegers.fromUnsignedByteArray("test2".getBytes())).publicImage
 
-    val height = r.h.bestFullBlockOpt.map(_.header.height).getOrElse(startHeight)
-    val boxToDoubleSpend: ErgoBox = r.h.bestFullBlockOpt.get.transactions.last.outputs.last
-    val input = Input(boxToDoubleSpend.id, ProverResult(Array.emptyByteArray, ContextExtension.empty))
+    val boxToDoubleSpend = r.h.bestFullBlockOpt.get.transactions.last.outputs.last
+    boxToDoubleSpend.proposition shouldBe defaultMinerSecret.publicImage
 
-    val outputs1 = IndexedSeq(new ErgoBoxCandidate(boxToDoubleSpend.value, prop1, creationHeight = height))
-    val tx1 = ErgoTransaction(IndexedSeq(input), outputs1)
+    val input = Input(boxToDoubleSpend.id, ProverResult(Array.emptyByteArray, ContextExtension.empty))
```

Comment:
We also `have emptyProverResult` in `WalletTestOps`. We can move it to `NodeViewBaseOps`